src/templates/react/Content/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+var path = require("path");
+var webpack = require("webpack");
+
+function resolve(filePath) {
+  return path.join(__dirname, filePath)
+}
+
+var babelOptions = {
+  presets: [["es2015", { "modules": false }]],
+  plugins: ["transform-runtime"]
+}
+
+module.exports = {
+  devtool: "source-map",
+  entry: resolve('./FableElmishReactTemplate.fsproj'),
+  output: {
+    filename: 'bundle.js',
+    path: resolve('./public'),
+  },
+  devServer: {
+    contentBase: resolve('./public'),
+    port: 8080
+  },
+  module: {
+    rules: [
+      {
+        test: /\.fs(x|proj)?$/,
+        use: {
+          loader: "fable-loader",
+          options: { babel: babelOptions }
```

Comment:
How do you set your env to be "production" ?
Seems like doing `node_modules/.bin/webpack-dev-server -p` isn't enough.

I am using `dotnet fable start` that's why I'm using webpack directly.